packages/cli/src/commands/composite/from-model.ts

```diff
@@ -18,6 +19,12 @@ export default class CompositeFromModel extends Command<Flags> {
       char: 'o',
       description: 'path to the file where the composite representation should be saved',
     }),
+    deploy: Flags.boolean({
+      char: 'd',
+      description:
+        'Deploy the composite to the ceramic node, which will start indexing on the composite',
+      default: true,
```

Comment:
If I'm reading the code correctly, I believe this is a behavior change.  I'm pretty sure `composite:from-model` does not index currently.  I think this should probably default to false.

Reply:
Good catch. That should default false.